api/python/provisioner/commands/set_swupgrade_repo.py

```diff
@@ -32,8 +32,74 @@
 
 class SetSWUpgradeRepo(SetSWUpdateRepo):
 
-    _REPO_DEST = PRVSNR_USER_FILES_SWUPGRADE_REPOS_DIR
     _BASE_DIR_PILLAR = "release/upgrade/base_dir"
+    _INSTALL_TYPE_PILLAR_KEY = "release/install_type"
+
+    def _configure_install_type(self, install_type: str) -> PillarKey:
```

Comment:
could double-check return value type declaration and what is actually should be returned?

Reply:
This code is no more needed for the newer approach. Deleted this method